src/components/TaskContainer.jsx

```diff
@@ -0,0 +1,251 @@
+      setIsShortDisabled(true);
+    } else {
+      setIsShortDisabled(false);
+    }
+    if (remainingTime < LONG_BREAK_MINUTES) {
+      setIsLongDisabled(true);
+    } else {
+      setIsLongDisabled(false);
+    }
+  }, [totalTasksTime]);
+
+  function deleteListItem() {
+    const currentTask = last(taskList);
+    const newTotalTasksTime = totalTasksTime - currentTask.duration;
+    setTotalTasksTime(newTotalTasksTime);
+
+    const updatedList = taskList.filter((task) => task.id !== currentTask.id);
+
+    setTaskList(updatedList ?? []);
+    setChangedAddBreakButton(false);
+    setShowBreakButton(false);
+
+    if (updatedList.length) {
+      const lastTask = last(taskList);
+      const lastUpdatedTask = last(updatedList);
+
+      setTitleChange(lastTask.title ?? '');
+      setMinutesChange(lastTask.duration ?? 0);
+
+      if (lastTask?.title === lastUpdatedTask?.title) {
+        setTaskAdded(true);
+      } else {
+        setTaskAdded(false);
+      }
+    }
+  }
+
+  function handleGetTotalScheduleTime(minutes) {
+    setTotalScheduleTime(minutes);
+  }
+
+  function handleChangedTimeInput() {
+    setChangedTimeInput(true);
+  }
+
+  function handleChangedAddTaskButton() {
+    const task = {
+      breakType: '',
+      duration: '',
+      id: uuid(),
+      title: '',
+      type: TASK_TYPES.TASK,
+    };
+
+    setChangedAddBreakButton(true);
+    setTaskList([...taskList, task]);
+    setTitleChange('');
+    setMinutesChange(DEFAULT_TASK_MINUTES);
+    setTaskAdded(false);
+  }
+
+  function handleChangedAddBreakButton() {
+    const lastTask = last(taskList);
+    const taskIndex = taskList.indexOf((task) => lastTask.id === task.id);
+    const duration = parseInt(minutesChange, 10);
+    const updatedTask = {
+      ...lastTask,
+      createdAt: new Date(),
+      duration,
+      title: titleChange,
+    };
+
+    taskList.splice(taskIndex, 1, updatedTask);
+    setShowBreakButton(true);
+
+    if (!isTaskAdded) {
+      setTotalTasksTime(totalTasksTime + duration);
+    }
+  }
+
+  function handleTaskTitleChange(event) {
+    setTitleChange(event.target.value);
+  }
+
+  function handleMinutesChange(event) {
+    setMinutesChange(event);
+  }
+
+  function handleChangedBreakDisplay(breakType) {
+    setShowBreakButton(false);
+    setChangedAddBreakButton(false);
+
+    const breakMinutes = breakType === SHORT_BREAK ? SHORT_BREAK_MINUTES : LONG_BREAK_MINUTES;
+    const task = {
+      breakType,
+      duration: breakMinutes,
+      id: uuid(),
+      title: breakType,
+      type: TASK_TYPES.BREAK,
+    };
+    setTaskList([...taskList, task]);
+
+    setTotalTasksTime(totalTasksTime + breakMinutes);
+  }
+
+  const list = taskList?.map((task) => (task.type === TASK_TYPES.TASK ? (
+    <Task
+      key={task}
+      handleMinutesChange={handleMinutesChange}
+      handleOnDelete={deleteListItem}
+      handleTaskTitleChange={handleTaskTitleChange}
+      minutesChange={minutesChange}
```

Comment:
A better name for this is `selectedMinute`

Reply:
Resolved